link/uprobe_multi.go

```diff
@@ -0,0 +1,164 @@
+package link
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"unsafe"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/asm"
+	"github.com/cilium/ebpf/internal"
+	"github.com/cilium/ebpf/internal/sys"
+	"github.com/cilium/ebpf/internal/unix"
+)
+
+// UprobeMultiOptions defines additional parameters that will be used
+// when opening a UprobeMulti Link.
+type UprobeMultiOptions struct {
```

Comment:
I see the kernel API has a `pid` argument, was that left out because not needed (or will be added in the future), or it doesn't work like normal uprobes' pid?

Reply:
ah it's leftover, I'll add it, it's the sae functionality as for uprobe, thanks

Reply:
This is still missing?

Reply:
ugh yep, sorry, will add it